python/CMakeLists.txt

```diff
@@ -0,0 +1,93 @@
+message(STATUS "Python version ${GEN_PYTHON_VERSION} was selected")
+unset(PYTHON_LIBRARY CACHE)
+unset(PYTHON_EXECUTABLE CACHE)
+unset(PYTHON_INCLUDE_DIR CACHE)
+
+if(${GEN_PYTHON_VERSION} STREQUAL "2")
+    find_package(PythonInterp 2 REQUIRED)
+    find_package(PythonLibs 2 REQUIRED)
+    if(NOT PYTHONLIBS_FOUND)
+        message(WARNING "Did not found Python version 2.x")
+        message(STATUS "Libyang supports Python 2.x and Python 3.x")
+    endif()
```

Comment:
So even Python 2 is supported?

Reply:
Yes since https://github.com/rjarry/libyang-cffi supports it. However, since Python 2 is officially EOL since the begining of 2020, I was not sure if you wanted to keep official Python 2 support.

Reply:
Only for some backwards compatibility, I guess, but it is better if @rkrejci decided this.